Makefile

```diff
@@ -19,21 +21,16 @@ help:
 
 .PHONY: dev
 dev: python
-	@tox -qe dev -- honcho start
+	@tox -qe dev
+
+.PHONY: supervisor
+supervisor: python
+	@tox -qe dev --run-command 'supervisorctl -c conf/supervisord-dev.conf $(command)'
 
 .PHONY: devdata
 devdata: python
 	@tox -qe dev -- python bin/devdata.py
 
-.PHONY: web
-web: python
-	@tox -qe dev
-
-.PHONY: nginx
-nginx: args?=up
-nginx: python
-	@tox -qe docker-compose -- $(args)
-
```

Comment:
When developing Via 3, it is necessary to run `nginx` separately from the web app. This is the same sort of feedback as the last PR in this area, so I won't go over all the reasons again.  But we should not remove a developer convenience, as this PR otherwise makes it strictly worse for me.

Reply:
I often want to run the web process (for example) separately to debug it. The way I've been doing this in lms and h is:

```
$ # Look in supervisor-dev.conf for the web process's command
~/P/via3 supervisor> cat conf/supervisord-dev.conf 
[supervisord]
nodaemon = true
silent = true

[program:web]
command=gunicorn -c conf/gunicorn/dev.conf.py --paste conf/development.ini
...

$ # Run the command copy-pasted from supervisor-dev.conf, in tox.
$ tox -qqe dev --run-command 'gunicorn -c conf/gunicorn/dev.conf.py --paste conf/development.ini'
[2020-08-10 13:55:32 +0100] [3577213] [INFO] Starting gunicorn 20.0.4
...
```

It helps if all of our apps use the same commands for the same processes.

Anyway, this has been working well for me and I think it might be the most reasonable solution. The problem with things like `make web` is that, previously on master:

1. `make dev` would run tox's `dev` environment
2. tox would run `honcho start` (or supervisor, now) inside the dev env
3. honcho would run `make web` and `make nginx`
4. Now our second instance of make (running inside tox, which is in turn running inside the first instance of make) would run the tox again
5. Now our second instance of tox (running inside a make that's running inside a tox that's running inside a make) would run `gunicorn`
6. A third instance of tox (in a make in a tox in a make) would then run `docker-compose` in another tox env

The convolutedness of this makes the dev env hard to understand (what happens when I run `make dev`?) and it also means that `make dev` has a very complex and deep process tree with multiple instances of make and tox running simultaneously and inside of each other.

This appears to have been the cause of multiple issue we've been having with `make dev` being slow to start up and shut down, messing up terminals after shutting down, leaving behind zombie processes and celery pid files etc, crashing when trying to autoreload the web server when the source code changes, etc.

It's difficult to pinpoint all or any of the above problems to one exact thing. They likely don't even have a single cause. One thing I can say is that they were _not_ caused by honcho: I was still able to reproduce the issues after moving from honcho to supervisor. So the general approach I took was "boil `make dev` down to the simplest possible process tree" and avoid running multiple instances of make or tox. Based on the last few weeks of problems this seems to have fixed the issues for h and lms.